### PR TITLE
Add text

### LINKED ISSUE #14: implement text types
from #3

### PR DESCRIPTION
**Pull request by marwahaha**
image

compare to figma: https://www.figma.com/file/PQD2GrejGoeZdYGhXthnAuEs/jupyter_master?node-id=974%3A5059
image

fixes #14 

cc @jameswenzel

I had to adjust the font-weight to get them to look similar to the spec.

### CODE REVIEW

File: src/components/H1Text/H1Text.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+const H1Text = styled.div`
```
Comment (jameswenzel) [inline src/components/H1Text/H1Text.tsx]: I think we should be using semantic HTML. Correct semantic HTML is an accessibility concern for screen readers, and apparently Google optimizes SEO at least partially based on semantic layout of the page:
https://medium.com/adalab/the-importance-of-semantic-html-78e74fb75ff0

**Comment by tgeorgeux** [comment]
@marwahaha Is the text in the image actually Helvetica Neue? Usually that font has a har flat line under the top portion of the 1 as is shown in the screenshot from Figma.

**Comment by marwahaha** [comment]
@tgeorgeux - good call out. I had reduced the font weight on the H1 and Subtitle 1 (and maybe others) because the font is much heavier than the one you have on Figma.

For example, here is H1 on my browser with font-weight: 400
image

**Comment by tgeorgeux** [comment]
Ah, for H1 did you set the font-weight to light?

**Comment by marwahaha** [comment]
@tgeorgeux - the font-weight does not have a `light` option. Here are the options:
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight

`lighter` is an option, but I don't think that's what you mean.

On Figma, do the font-weights refer to this page? Light=300, which does not have the flat line under the 1.
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Common_weight_name_mapping

**Comment by marwahaha** [comment]
@tgeorgeux  - merging now. If there are problems we can revisit in a new PR.

**Pull request merged**